src/pages/en/tutorial/1-setup/1.mdx

```diff
@@ -26,15 +26,22 @@ You will use a **command line (terminal)** to create your Astro project and to r
 
 You can access the command line through a local terminal program for your operating system. Common applications include **Terminal** (MacOS/Linux), **Command Prompt** (Windows), and **Termux** (Android). One of these will probably already be on your machine. 
 
-For Astro to run on your system, you will also need to have [**Node.js**](https://nodejs.org/en/) installed. To check to see whether you already have a compatible version (`v16.12.0` or higher) installed: 
+### Node.js
 
-1. Launch your terminal and type the following into the command line:
+For Astro to run on your system, you also need to have [**Node.js**](https://nodejs.org/en/) of version `v16.12.0` or later.
+First, check whether you already have Node.js installed and check the installed version. To do this, run the following command in your terminal:
 
-    ```sh
-    node -v
-    ```
+```sh
+node -v
 
-2. If necessary, follow the instructions to [install Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) via the command line.
+// Example output
+v16.14.0
+```
+
+* If the command returns a version number higher than `v16.12.0`, you're good to go! 🎉 If not, you need to install the compatible Node.js version.
+* If the command returns an error message like `Command 'node' not found`, Node.js is not installed in your system.
```

Comment:
```suggestion
If the command returns a version number higher than `v16.12.0`, you're good to go!

If the command returns an error message like `Command 'node' not found`, or a version number lower than `v16.12.0`, then you need to [install a compatible Node.js version](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
```
I think it makes more sense to split these into a success/failure case, vs either being good or not in the first point, then not being good in the second point. And in that case, we can bring the link up into this point with the solution, and avoid the next paragraph entirely!  🥳 

Also, bullet points don't seem quite right here? We typically only use them to finish sentences with multple options, or a distinct list of comparable items. So, I could see using them if we did something like:

> If the command returns: 
> - a version number higher than...
> - an error message or a version number lower than...

You could try that and see if you think it looks good!

Reply:
> I think it makes more sense to split these into a success/failure case, vs either being good or not in the first point, then not being good in the second point. And in that case, we can bring the link up into this point with the solution, and avoid the next paragraph entirely! 🥳

Well, yes, agreed. As long as the solution is the same for both cases, we can do that trick :)